android/src/main/kotlin/com/rudderstack/sdk/kotlin/android/plugins/devicemode/IntegrationPlugin.kt

```diff
@@ -216,17 +217,14 @@ abstract class IntegrationPlugin : EventPlugin {
     }
 
     private fun applyDefaultPlugins() {
+        add(EventFilteringPlugin(key))
```

Comment:
NIT: I would suggest wrapping the addition of custom integration plugins (standard integration plugins may not be necessary as we are maintaining them) in a try/catch block to ensure there's no event loss even when some issue occurs during the `setup` process!

We can consider doing this in future, but I thought to highlight it now!